scripts/install_hpfeeds.sh

```diff
@@ -58,7 +58,7 @@ ldconfig /usr/local/lib/
 mkdir -p /opt
 cd /opt
 rm -rf /opt/hpfeeds
-git clone https://github.com/threatstream/hpfeeds
+git clone https://github.com/Antelox/hpfeeds
```

Comment:
Once we merge the hpfeeds change this change won't be necessary.  This change needs to be reverted.  since we would merge all of these at the same time.